lib/compiler.js

```diff
@@ -1,5 +1,5 @@
 var assert = require('assert');
-var cloneDeep = require('lodash.clonedeep');
+var cloneDeep = require('lodash/lang/cloneDeep');
```

Comment:
Good point. I'll leave it as is though in case anyone reading gets curious about it and then accidentally learns something cool about lodash. ;-)